android/quest/src/test/java/org/smartregister/fhircore/quest/ui/profile/ProfileViewModelTest.kt

```diff
@@ -200,10 +203,33 @@ class ProfileViewModelTest : RobolectricTest() {
     every { listActionConfig.handleClickEvent(navController, resourceData) } just runs
     profileViewModel.onEvent(event)
     every { mockProfileViewModel.onEvent(any()) } just runs
-
     verifyAll {
       mockActionConfig.interpolate(any())
       profileViewModel.onEvent(event)
     }
   }
+
+  @Test
+  fun testThatChangeManagingEntityLogsWarningWhenItIsNull() {
+    mockkObject(Timber.Forest)
+    every { Timber.w(any<String>()) } just runs
+    val overflowMenuItemConfig =
+      OverflowMenuItemConfig(
+        id = 1,
+        title = "myFlowMenu",
+        confirmAction = false,
+        icon = null,
+        titleColor = BLACK_COLOR_HEX_CODE,
+        backgroundColor = null,
+        visible = "true",
+        showSeparator = false,
+        enabled = "true",
+        actions = emptyList(),
+      )
+    val navController = mockk<NavController>()
+    val event = ProfileEvent.OverflowMenuClick(navController, resourceData, overflowMenuItemConfig)
+    val mockManagingEntity: ManagingEntityConfig? = null
+    profileViewModel.changeManagingEntity(event, mockManagingEntity)
+    verify { Timber.w("ManagingEntityConfig required. Base resource should be Group") }
```

Comment:
Can we add extra verification/assertion here other than just verifying the log? the log here is subject to change. Verify/assert something that affects the `profileViewModel.changeManagingEntity` functionality instead.

Reply:
Yes sure working on it